### PR TITLE
CToken delegate for a comp-like token

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

**Review by hayesgm — COMMENTED** [review]
Clean and simple

File: tests/Tokens/compLikeTest.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  makeCToken,
+} = require('../Utils/Compound');
+
+
+describe('CCompLikeDelegate', function () {
+  describe("_delegateCompLikeTo", () => {
+    it("does not delegate if not the admin", async () => {
+      const [root, a1] = saddle.accounts;
+      const cToken = await makeCToken({kind: 'ccomp'});
+      await expect(send(cToken, '_delegateCompLikeTo', [a1], {from: a1})).rejects.toRevert('revert only the admin may set the comp-like delegate');
```
Comment (hayesgm) [inline tests/Tokens/compLikeTest.js:11]: This works due to the fallback in `CErc20Delegator`, right?

**Review by hayesgm — APPROVED** [review]
looks great. Thanks for changing the delegator code.

File: tests/Tokens/compLikeTest.js (hunk 1 above)
Comment (jflatow) [reply tests/Tokens/compLikeTest.js:11]: Yeah it delegates everything

**Pull request merged**